frontend/src/app/components/team/team.component.html

```diff
@@ -1,18 +1,23 @@
-<div *ngIf="overviewEntity as OVEntity" class="bg-grey justify-content-between pt-3">
+<div *ngIf="overviewEntity as OVEntity" class="bg-color justify-content-between pt-3">
   <section class="team-header">
     <div class="d-flex align-items-center gap-2 flex-row">
-      <h1 class="team-title">{{ OVEntity.team.name }}</h1>
+      <h1 id="overview-team-title" class="team-title">{{ OVEntity.team.name }}</h1>
     </div>
     <button
       (click)="createObjective()"
       *ngIf="OVEntity.writeable"
+      id="add-objective"
       [attr.data-testId]="'add-objective'"
       class="px-0 mt-1 mb-3 pe-2 h-50 align-new-icon"
       color="primary"
       mat-button
     >
-      <span class="d-flex align-items-center add-text">
-        <img alt="Add key-result button" class="add-cross-button" src="../../../assets/icons/new-icon.svg" />
+      <span id="add-objective-span" class="d-flex align-items-center add-text">
+        <img
+          alt="Add key-result button"
+          class="add-cross-button"
+          src="{{ (addIconSrc | async) ?? '/assets/icons/new-icon.svg' }}"
```

Comment:
```suggestion
          src="{{ (addIconSrc | async)  }}"
```
Remove from here ...